go.mod

```diff
@@ -37,7 +37,7 @@ require (
 	github.com/google/go-cmp v0.3.1
 	github.com/google/gofuzz v1.1.0
 	github.com/google/gopacket v1.1.17
-	github.com/google/gops v0.3.6
+	github.com/google/gops v0.3.8
```

Comment:
If this dep is bumped, I think we also want to bump gops binaries (eg [here](https://github.com/cilium/cilium/blob/fb8481a2e9e0f006dde144f64b196addc5fe8529/hubble-relay.Dockerfile#L12) for hubble-relay).

Reply:
Good point, forgot about this. Since the vendoring bump is not strictly necessary for this and I'm waiting for an additional upstream PR to be merged (google/gops#104), I'll take the vendoring update out of this PR and send a separate PR for it. This will hopefully also make it easier to backport.